ImmuDB4Net.Tests/BasicClientIntegrationTests.cs

```diff
@@ -132,6 +132,25 @@ public async Task Test2()
             CollectionAssert.AreEqual(entry.Value, values[i]);
         }
 
+        await client.Close();
+    }  
+    
+    [TestMethod("execute sqlexec and query")]
+    public async Task Test3()
+    {
+        await client!.Open("immudb", "immudb", "defaultdb");
+        var rspCreate = await client.SQLExec("CREATE TABLE IF NOT EXISTS logs(id INTEGER AUTO_INCREMENT, created TIMESTAMP, entry VARCHAR, PRIMARY KEY id)");        
+        Assert.AreEqual(1, rspCreate.Items.Count);
+        var rspIndex = await client.SQLExec("CREATE INDEX IF NOT EXISTS ON logs(created)");
+        Assert.AreEqual(1, rspIndex.Items.Count);
+        var rspInsert = await client.SQLExec("INSERT INTO logs(created, entry) VALUES($1, $2)", 
+            new SQL.SQLParameter(DateTime.UtcNow),
+            new SQL.SQLParameter("entry1"));
+        var queryResult = await client.SQLQuery("SELECT created, entry FROM LOGS order by created DESC");
+        Assert.AreEqual(2, queryResult.Columns.Count);
+        Assert.AreEqual(1, queryResult.Rows.Count);
+        var sqlVal = queryResult.Rows[0]["(defaultdb.logs.entry)"];

```

Comment:
queryResult object could provide a higher level of abstraction to consume returned row data.
the column encoding `(defaultdb.logs.entry)` could be hidden and it might be possible to read the value just by using the column name

Reply:
done